chord/chord.go

```diff
@@ -45,6 +45,7 @@ type ChordNode struct {
 	// Note: it must not be changed after initial definition in NewChordNode()
 	node Node
 
+	next    	   nextWithMux
```

Comment:
Each node should have a `next` index used by fix_finger _(next stores the index of the next finger to fix.)_